web/sites/default/example.services.config.yml

```diff
@@ -0,0 +1,10 @@
+# If you want to activate file-based configuration with config_suite:
+# copy this file "services.config.yml".
+services:
+  # File-based configuration.
```

Comment:
This is cool -- is it still using config_suite in reality? (Meaning, this isn't doing the thing we used to do with site builds where the config IS stored in files and never in the database, is it?)

Reply:
yes! I have config_suite installing on socalgrantmakers, the first build, and I have [this PR in on kmc](https://github.com/United-Philanthropy-Forum/km-collaborative/pull/497).

Reply:
Well nice! I see you're assigned to a couple of tickets related to this in our dev-team board.

https://github.com/thinkshout/dev-team/issues/25
and 
https://github.com/thinkshout/dev-team/issues/34

What are next steps so that we can move to this method on other projects, assuming that we do want to do that?

I'll merge this.

Reply:
Maybe the first step is to show you how it's behaving and get some confirmation/feedback ;) Then clean up any loose ends and do that documentation piece, create a PR for drupal-project, and look at converting any existing site that doesn't like its config management situation as an example to use for documenting change-over?

honestly, I'd love to get a clearer idea of how different this is from our other setups. I mostly work with those assuming the magic you put in place works as expected, and it did ;)

Reply:
Cool, I'm inviting you to a meeting to talk about this at the next guild gatherings timeslot.